### DISCUSSION TITLE
Is it possible to addChild to a group node in a specific react component div ?

### DISCUSSION THREAD

**Discussion by yahiaouik** [antvis/X6#1531] [Q&A] [answered]
I'm using a react component to render a group node. 
```
<div className="GroupContainer">
  <div className="Header">
  </div>
  <div className="Body">
  </div>
</div>
```

I would like to addChild into the Body div of the component. 

By default, the addChild function add children into the global foreinObject (the global component)  as it is children can be attach to the header div.

Is there any way to do it ?

**Comment by NewByVector** [comment, marked as answer]
AddChild does not make one node part of another, but has the ability to move the parent node and the child node will moved together.
